src/dg/dg.cpp

```diff
@@ -2095,7 +2129,9 @@ void DGBase<dim,real,MeshType>::allocate_system (
 
     dof_handler.distribute_dofs(fe_collection);
     //This Cuthill_McKee renumbering for dof_handlr uses a lot of memory in 3D, is there another way?
-    if(all_parameters->do_renumber_dofs) dealii::DoFRenumbering::Cuthill_McKee(dof_handler,true);
+    if(all_parameters->renumber_dof_handler_Cuthill_Mckee && all_parameters->do_renumber_dofs){
```

Comment:
Just to confirm, `all_parameters->do_renumber_dofs` renumbers solution dofs while `all_parameters->renumber_dof_handler_Cuthill_Mckee` renumbers dofs of both solution and grid coeffs? I guess we currently cannot renumber just the solution dofs (we can probably implement it in future if required).

Reply:
Yes. In future should have dof_handler specific parameters where can have the flags do_renumber_dofs and do_renumber_dofs_grid, then choose specifically what method to renumber the dofs with.

Reply:
Could you add comments to `all_parameters` indicating the difference between these two parameters as they currently are implemented? They are named so similarly that it could be a source of confusion!

Reply:
This will be addressed by issue #220

Reply:
you could even just add the enum and set `renumber_dof_handler_Cuthill_Mckee` in `al_parameters.cpp` based on `do_renumber_dofs` and `renumber_type`

Reply:
Fixed